app/organisation/invite_rest.py

```diff
@@ -106,3 +112,32 @@ def invited_org_user_url(invited_org_user_id, invite_link_host=None):
         invite_link_host = current_app.config['ADMIN_BASE_URL']
 
     return '{0}/organisation-invitation/{1}'.format(invite_link_host, token)
+
+
+@organisation_invite_blueprint.route('/invite/organisation/<uuid:invited_org_user_id>', methods=['GET'])
```

Comment:
What do you think about migrating these routes to be more consistent with each other? I'm not sure if that matters much, although it may be confusing to someone reading this in future as to why they have different structures.

Reply:
sorry, I don't follow. Which route should it be more consistent with? The ones above where it's `/service/<uuid>/invite/<uuid>`?

We don't have a service ID here so cannot do that.

Reply:
You're right they can't have exactly the same structure. I was looking at the beginning:

- Some routes start with `/invite/organisation`
- Others start with `/organisation`

I'd (perhaps naively) expect all the routes in a file to at least start with the same thing, which in this file should probably be `/invite/organisation`. I think it's a minor one, though, as we don't seem to have any guidance on URL structure. Still, this might be a good opportunity to make them consistent, since we're having to do change some of the other routes anyway. Up to you, though - I don't feel strongly about it.

Reply:
ah, gotcha. i see what you mean, but if we do that we sacrifice clarity across different organisation endpoints. 

for example we currently have:

```
GET        /organisations/<uuid:organisation_id>  # get organisation info
GET        /organisations/<uuid:organisation_id>/services  # get info on services for organisation
GET        /organisation/<uuid:organisation_id>/invite  # get info on invites for organisation
```

(please ignore the pluralisation of organisation aaaaaagh 🐰🕳️😱)

think this is a six/half dozen problem, we're never going to be able to align invites (both with a ID and without ID) and also align organisation endpoints at the same time. i'll leave it as is to avoid extra work for myself

Reply:
Sure, no worries. I can only sympathise 😸.